cmd/api/src/database/migration/agi.go

```diff
@@ -70,6 +65,20 @@ func (s *Migrator) updateAssetGroups() error {
 			}
 		}
 
+		if _, hasOwned := systemAssetGroups.FindByName(model.OwnedAsserGroupTag); !hasOwned {
+			log.Infof("Missing the default Owned asset group. Creating it now.")
+
+			ownedAG := model.AssetGroup{
+				Name:        model.OwnedAssetGroupName,
+				Tag:         model.OwnedAsserGroupTag,
+				SystemGroup: true,
+			}
+
+			if result := tx.Create(&ownedAG); result.Error != nil {
+				return result.Error
+			}
```

Comment:
Since we aren't doing anything with the error, we can actually shortcut this bit:

```go
return tx.Create(&ownedAG).Error
```